cli/client.go

```diff
@@ -292,6 +295,52 @@ func (c *AppClient) selectOrganization(orgStr string) error {
 	return nil
 }
 
+// GetOrg gets an org by an indentifying string. If the orgStr is an
+// org UUID, then this matchs on organization ID, otherwise this will match
+// on organization name.
+func (c *AppClient) GetOrg(orgStr string) (*apppb.Organization, error) {
```

Comment:
This takes the internals of `selectOrganization` and makes them stateless rather than changing `c.selectedOrganization`. It results in much cleaner code for this use case